annotations.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	mkr "github.com/mackerelio/mackerel-client-go"
+	"github.com/mackerelio/mkr/logger"
+	"gopkg.in/urfave/cli.v1"
+	"os"
+)
+
+var commandAnnotations = cli.Command{
+	Name: "annotations",
+	Subcommands: []cli.Command{
+		{
+			Name:        "create",
+			Usage:       "create annotation",
+			Description: "Creates an annotation.",
+			Action:      doAnnotationsCreate,
+			Flags: []cli.Flag{
+				cli.StringFlag{Name: "title", Value: "", Usage: "Title for annotation."},
+				cli.StringFlag{Name: "description", Value: "", Usage: "Description for annotation."},
+				cli.IntFlag{Name: "from"},
+				cli.IntFlag{Name: "to"},
+				cli.StringFlag{Name: "service, s", Value: "", Usage: "Service name for annotation."},
+				cli.StringSliceFlag{
+					Name:  "role, r",
+					Value: &cli.StringSlice{},
+					Usage: "Roles for annotation. Multiple choices are allowed.",
+				},
+			},
+		},
+		{
+			Name:        "list",
```

Comment:
Hmm... I like "list", but t's "find" in mackerel-client-go https://github.com/mackerelio/mackerel-client-go/blob/master/graph_annotation.go#L30 and it's "obtain" in document https://mackerel.io/api-docs/entry/graph-annotations#get

I prefer consistent naming. I'm :+1: to "list", but maybe it's ok anyway.